src/components/Menu/Menu.js

```diff
@@ -65,38 +65,55 @@ class Menu extends Component {
    * render
    * @return {XML}
    */
+
   render() {
-    const { type } = this.props;
+    const { type, campaign, baseUrl } = this.props;
+    const { menuItems } = this.state;
 
-    if (this.state.menuItems.length >= 1) {
+    if (menuItems.length >= 1) {
       return (
         <nav className="menu--footer">
           <ul className="menu" id={`${type}-menu`}>
-            {this.state.menuItems.map((item) => {
-              if (this.props.campaign === 'sportrelief') {
+            {menuItems.map((item) => {
+              const url = item.link ? item.link.url : item.url;
+              const title = item.link ? item.link.title : item.title;
+
+              if (campaign === 'sportrelief') {
                 return (
-                  <li className="menu-item">
-                    {(item.link.url.indexOf('http') !== -1) ?
-                      <a href={item.link.url} rel="noopener noreferrer" target="_blank">{item.link.title}</a> :
-                      <a href={this.props.baseUrl + item.link.url}>{item.link.title}</a>}
+                  <li className="menu-item" key={`${type}-menu-${title}`}>
+                    {(url.indexOf('http') !== -1) ?
+                      <a href={url} rel="noopener noreferrer" target="_blank">{title}</a> :
+                      <a href={baseUrl + url}>{title}</a>}
                   </li>
                 );
               }
 
               return (
-                <li className="menu-item">
-                  {(item.url.indexOf('http') !== -1) ?
-                    <a href={item.url} rel="noopener noreferrer" target="_blank">{item.title}</a> :
-                    <a href={item.url}>{item.title}</a>}
+                <li className="menu-item" key={`${type}-menu-${title}`}>
+                  {(url.indexOf('http') !== -1) ?
+                    <a href={url} rel="noopener noreferrer" target="_blank">{title}</a> :
+                    <a href={url}>{title}</a>}
                 </li>
               );
             })}
           </ul>
         </nav>
       );
     }
-
-    return <div />;
+    return (
+      <nav className="menu--footer">
+        <ul className="menu" id={`${type}-menu`}>
+          <div>
+            <li className="menu-item">
+              <a href={campaign === 'sportrelief' ? 'https://lite.sportrelief.com/terms-of-use' : 'https://lite.comicrelief.com/legal/'} rel="noopener noreferrer" target="_blank">Legal</a>
```

Comment:
Nearly there man, we just need to be able to set these via props (as an object, i guess), so it's flexible enough to use anywhere